src/Consumer.php

```diff
@@ -39,12 +39,22 @@ public function consume($queue, Closure $closure)
                 no_local: Don't receive messages published by this consumer.
                 no_ack: Tells the server if the consumer will acknowledge the messages.
                 exclusive: Request exclusive consumer access, meaning only this consumer can access the queue
-                nowait:
+                nowait: basic_qos() method is using to tell RabbitMQ not to give more than one message to a worker at a time.
```

Comment:
```suggestion
                nowait:
                basic_qos: The consumer-prefetch make it possible to limit the number of unacknowledged messages on a channel (or connection) when consuming.
```
I might fix the explaination for nowait in a different PR.